### PR TITLE
Pre-emptive cleanup of lazy_relay test.

### PR DESCRIPTION
**Pull request by Raynos**
This fixes a bunch of test that didnt whitelist their logs.

r: @jcorbin @rf

### CODE REVIEW

File: test/relay_lazy.js [hunk 1]
```diff
@@ -363,14 +376,25 @@ allocCluster.test('relay request declines on no peer', {
...
+
+        assert.equal(relayLine.meta.hostPort, relay.hostPort);
+        assert.equal(relayLine.meta.remoteName, source.hostPort);
+
         assert.end();
     }
 });
 
-allocCluster.test('relay request handles channel close correctly', {
+allocCluster.test.only('relay request handles channel close correctly', {
```
Comment (rf) [inline test/relay_lazy.js]: only?

File: relay_handler.js [hunk 2]
```diff
@@ -242,6 +242,10 @@ LazyRelayInReq.prototype._extendLogInfo =
 function _extendLogInfo(info) {
     var self = this;
 
+    if (self.conn) {
```
Comment (rf) [inline relay_handler.js]: wasn't this in a different PR?

File: test/relay_lazy.js (hunk 1 above)
Comment (Raynos) [inline test/relay_lazy.js]: FFFFFFFFFFFF----

File: relay_handler.js (hunk 2 above)
Comment (Raynos) [inline relay_handler.js]: Yes; but needed in this one too.

**Pull request merged**